.github/workflows/gh-linux_mac.yml

```diff
@@ -0,0 +1,360 @@
+#            mkdir build && cd build
+#            cmake -DCMAKE_PREFIX_PATH=$BUILD_REPOSITORY_LOCALPATH/dist .. && make
+#            ./ExampleExe
+#            if [[ "TILEDB_STATIC" == "ON" ]]; then
+#              ./ExampleExe_static
+#            fi
+#            popd
+#            # Build and run the PNG ingestion example.
+#            if [[ "$AGENT_OS" == "Linux" ]]; then
+#              # libpng (example dependency)
+#              sudo apt-get install libpng-dev
+#
+#              pushd $BUILD_REPOSITORY_LOCALPATH/examples/png_ingestion;
+#              mkdir build && cd build;
+#              cmake -DCMAKE_PREFIX_PATH=$BUILD_REPOSITORY_LOCALPATH/dist .. && make;
+#              ./tiledb_png $BUILD_REPOSITORY_LOCALPATH/doc/source/figures/Scarlet-Macaw-small.png /tmp/pngarray output.png;
+#              popd;
+#            fi
+#          fi
+#
+#          # Build the benchmarks (but do not run them).
+#          pushd $BUILD_REPOSITORY_LOCALPATH/test/benchmarking && \
+#            mkdir build && cd build && \
+#            cmake -DCMAKE_PREFIX_PATH=$BUILD_REPOSITORY_LOCALPATH/dist ../src && make && \
+#          popd
+#
+#          testfile=$(mktemp)
+#          mv $testfile $testfile.cc
+#          testfile=$testfile.cc
+#          cat <<EOF > $testfile
+#          #include <assert.h>
+#          #include <tiledb/tiledb.h>
+#          #include <tiledb/version.h>
+#          int main(int argc, char **argv) {
+#            int major = 0;
+#            int minor = 0;
+#            int patch = 0;
+#            tiledb_version(&major,&minor,&patch);
+#            auto version = tiledb::version();
+#            assert(major == std::get<0>(version));
+#            return 0;
+#          }
+#          EOF
+#          export TESTFILE_LDFLAGS="-ltiledb"
+#          if [[ "$AGENT_OS" == "Linux" ]]; then
+#            export LD_LIBRARY_PATH=$BUILD_REPOSITORY_LOCALPATH/dist/lib:/usr/local/lib:$LD_LIBRARY_PATH
+#          else
+#            export DYLD_LIBRARY_PATH=$BUILD_REPOSITORY_LOCALPATH/dist/lib:/usr/local/lib:$DYLD_LIBRARY_PATH
+#          fi
+#          $CXX -std=c++11 -g -O0 -Wall -Werror -I$BUILD_REPOSITORY_LOCALPATH/dist/include -L$BUILD_REPOSITORY_LOCALPATH/dist/lib $testfile -o $testfile.exe $TESTFILE_LDFLAGS && \
+#          $testfile.exe && \
+#          rm -f $testfile $testfile.exe
+#
+#          ps -U $(whoami) -o comm= | sort | uniq
+#          #  displayName: 'Build examples, PNG test, and benchmarks (build-only)'
+
+      - name: 'Test status check'
+        run: |
+          # tiledb_unit is configured to set a job-level variable TILEDB_CI_SUCCESS=1
+          # following the test run. If this variable is not set, the build should fail.
+          # see https://github.com/TileDB-Inc/TileDB/pull/1400 (5f0623f4d3)
+          if [[ "${{ steps.test.outputs.TILEDB_CI_SUCCESS }}" -ne 1 ]]; then
+            exit 1;
+          fi
+#
+      - name: "Print log files (failed build only)"
```

Comment:
Have you verified that this works w/ a failing build? It can be pretty useful for debugging b/c it's the only way to get the context of cmake sub-commands and openssl configure etc.

Reply:
Definitely it does the job